docs/tips/js/syntax-highlighting.md

```diff
@@ -5,19 +5,23 @@ In order to enable syntax highlighting, you must use a client-side JavaScript hi
 ```yaml
 page:
   headHtml: |
-    <link href="https://cdn.jsdelivr.net/npm/prismjs@1.23.0/themes/prism-tomorrow.css" rel="stylesheet" />
-    <script src="https://cdn.jsdelivr.net/combine/npm/prismjs@1.23.0/prism.min.js,npm/prismjs@1.23.0/plugins/autoloader/prism-autoloader.min.js"></script>
-```
-
-Or, using the alias from the default layer's `index.yaml`:
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.28.0/themes/prism-tomorrow.min.css" rel="stylesheet" />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.28.0/prism.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.28.0/plugins/autoloader/prism-autoloader.min.js"></script>
```

Comment:
Any reason we can't use `<snippet var="js.prism" />` here?

Reply:
Was unsure if we should keep the non-snippet variants.

Should I remove the non-snippet variants for js.prism & js.mermaid docs?

Reply:
I've changed to only suggest the `<snippet var="..." />` instead